component-samples/demo/README.MD

```diff
@@ -0,0 +1,233 @@
+# Table of Contents
+1. [System Requirements](#system-requirements)
+2. [Docker Commands](#docker-commands)
+3. [Podman Commands](#podman-commands)
+4. [Running Demo](#running-demo)
+5. [ServiceInfo Setup](#serviceinfo-setup-between-fdo-pri-http-java-device-sample-and-fdo-pri-owner-sample)
+6. [Enabling ServiceInfo transfer](#enabling-serviceinfo-transfer)
+7. [Customize for multi machine setup](#)
+
+# System Requirements
+
+* Operating System: Ubuntu 20.04 / RHEL 8.4
+
+* Linux packages:
+
+  - Docker engine (minimum version 18.09, Supported till version 20.10.7) / Podman engine (For RHEL)
+
+  - Docker-compose (minimum version 1.21.2) / Podman-compose (For RHEL)
+
+  - Haveged
+
+  - Java 11
+
+  - Maven (minimum 3.6.3)
+  
+
+# Docker Commands
+
+## Start Docker
+* Use the following command to start the docker container.
+```
+sudo docker-compose up -d --build
+```
+
+## Stop Docker
+
+* Use the following command to stop a specific docker container.
+```
+sudo docker stop <container-name>
+```
+OR
+```
+sudo docker stop <container-id>
+```
+
+* Use the following command to stop all running docker containers.
+```
+sudo docker stop $(sudo docker ps -a -q)
+```
+
+## Clean up Containers
+
+* Use the following command to remove a specific container.
+```
+sudo docker rm <container-name>
+```
+OR
+```
+sudo docker rm <container-id>
+```
+
+* Use the following command to remove the docker image.
+```
+sudo docker rmi <image-name>
+```
+OR
+```
+sudo docker rmi <image-id>
+```
+
+* Use the following command to delete all the docker artifacts. (**Note:** Docker containers must be stopped before deleting them)
+```
+sudo docker system prune -a
+```
+
+# Podman Commands
+
+## Start Podman
+* Use the following command to start the podman container.
+```
+podman-compose up -d --build
+```
+
+## Stop Podman
+
+* Use the following command to stop a specific podman container.
+```
+podman stop <container-name>
+```
+OR
+```
+podman stop <container-id>
+```
+
+* Use the following command to stop all running podman containers.
+```
+podman stop -a
+```
+
+## Clean up Containers
+
+* Use the following command to remove a specific container.
+```
+podman rm <container-name>
+```
+OR
+```
+podman rm <container-id>
+```
+
+* Use the following command to remove the podman image.
+```
+podman rmi <image-name>
+```
+OR
+```
+podman rmi <image-id>
+```
+
+* Use the following command to delete all the podman artifacts. (**Note:** podman containers must be stopped before deleting them)
+```
+podman system prune -a
+```
+# Configuring Proxies
+Update the proxy information in `_JAVA_OPTIONS` as
```

Comment:
Should we consolidate all proxy information in one place?
Also, lets talk about this readme and its path